buildAndReleaseTask/writeprovenance.ts

```diff
@@ -63,7 +63,7 @@ export async function writeArtifactJson (subjectJson: Array<subjectArtifact>,
               "buildFinishedOn": new Date()
             },
             "recipe": {
-              "type": "https://dev.azure.com/Attestations/ProvenanceGenerator@0",
+              "type": "https://dev.azure.com/Attestations/SLSAProvenanceGenerator@0",
```

Comment:
The `recipe` should be the type of build recipe, so should this be something like "AzurePipeline"? (I'm not familiar with Azure DevOps.)

Reply:
So in this context, "SLSAProvenanceGenerator@0" is the name of the Azure Pipeline plugin that is doing the work.  This name is immutable and someone else on the platform (globally) can't use it.

Reply:
Let's submit this PR as-is since it's a clear improvement. I'll file a separate issue regarding how the fields are populated.

Reply:
ack